scripts/cmaq_preprocess/bias_correct_cams.py

```diff
@@ -17,14 +18,18 @@ def main():
     icon_file = get_icon_file(config)
     bcon_files = get_bcon_files(config)
     met_file = get_met_file(config)
-
+    if int(os.getenv('CORRECT_BIAS_BY_REGION','1')) > 0:
+        correct_bias_by_region = True
+    else:
+        correct_bias_by_region = False
```

Comment:
It seems that we're retaining the previous behaviour and enabling the "fixed" behaviour using an environment variable.

My first question is, what's the value in preserving the old behaviour if it's known to be problematic?

Second question is, have we got the correct default here? Is the fixed behaviour "on by default" or will we need to add some configuration to our infra for this to take effect in prod?